app/cdap/components/PipelineConfigurations/PipelineConfigurations.scss

```diff
@@ -22,6 +22,10 @@ $top-panel-height: 54px;
 $right-position: -360px;
 $sidepanel-width: 170px;
 
+.pipeline-config-modal {
+  z-index: 1060; // overriding default 1300 to make nested modals (1061) visible.
```

Comment:
we need to figure out a better long term solution from this in the future @njbriggs

Reply:
One solution could be having a JS counter variable and increment/decrement as we open/hide modals.